rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/QrdaControllerV1.java

```diff
@@ -68,16 +69,29 @@ public QrdaControllerV1(final QrdaService qrdaService, final ValidationService v
 
 		validationService.validateQpp(conversionReport);
 
-		auditService.success(conversionReport);
+		Metadata metadata = audit(conversionReport);
 
 		API_LOG.info("Conversion request succeeded");
 
 		HttpHeaders httpHeaders = new HttpHeaders();
 		httpHeaders.setContentType(MediaType.APPLICATION_JSON_UTF8);
 
+		if (metadata != null) {
+			httpHeaders.add("Location", metadata.getUuid());
```

Comment:
Please choose a different response header name.  `Location` is a standard response header field and we don't want to override the definition of what that is.  See: https://en.wikipedia.org/wiki/List_of_HTTP_header_fields

Reply:
So, I thought it was actually good to use `Location`.  We return a HTTP status 201 Created which should include a `Location` header, but we don't.  This PR now makes us use the `Location` header.  Granted, it isn't a URL but just a UUID, but I thought it was the closest thing we've got.  We can't use the CPC+ API.

Reply:
Hmm... yeah, I agree that it's the closest match.  ETag is close as well, but the ETag should change if the resource is modified, which it won't.

Perhaps it's best to just use Location, then.